okapi-core/src/test/java/org/folio/okapi/MainDeployTest.java

```diff
@@ -22,30 +22,28 @@
 public class MainDeployTest {
 
   private final Logger logger = OkapiLogger.get();
-  private final int port = 9230;
-  private Async async;
-  private Vertx vertx;
-  private RamlDefinition api;
+  private static final int port = 9230;
+  // private Async async;
+  private static RamlDefinition api;
 
-  @Before
-  public void setUp(TestContext context) {
+  @BeforeClass
+  public static void setupBeforeClass(TestContext context) {
     System.setProperty("vertx.logger-delegate-factory-class-name",
         "io.vertx.core.logging.Log4jLogDelegateFactory");
     // can't set Verticle options so we set a property instead
     System.setProperty("port", Integer.toString(port));
-    async = context.async();
     api = RamlLoaders.fromFile("src/main/raml").load("okapi.raml");
     RestAssured.port = port;
-    async.complete();
+  }
+
+  @Before
+  public void setup(TestContext context) {
+    System.setProperty("port", Integer.toString(port));
   }
 
   @After
   public void tearDown(TestContext context) {
     System.setProperty("port", ""); // disable port by emptying it
-    if (vertx != null) {
-      vertx.close(context.asyncAssertSuccess());
-      vertx = null;
-    }
```

Comment:
`@After` can be changed to `@AfterClass`, setProperty can be changed to clearProperty.

Reply:
yep. And @Before can be removed entirely.